.github/workflows/ci.yml

```diff
@@ -99,10 +100,11 @@ jobs:
       - name: Set up Environment
         if: github.event_name != 'release'
         run: |
-            echo "CIBW_SKIP=${CIBW_SKIP} *-musllinux_* cp38-*_aarch64 cp39-*_aarch64 cp311-*_aarch64 cp312-*_aarch64" >> $GITHUB_ENV
+            echo "CIBW_BUILD=${CIBW_BUILD} cp313-*" >> $GITHUB_ENV
```

Comment:
For the moment 3.13 wheels will only be built for non-release workflows.

Reply:
☝️ Outdated with the release of Python 3.13.0rc1 since we want to start publishing `cp313-*` wheels now.